wiki/tests/testdata/models.py

```diff
@@ -8,3 +8,7 @@ class CustomUser(AbstractUser):
         some_field = models.IntegerField(default=0)
 except ImportError:
     pass
+
+
+class CustomGroup(models.Model):
```

Comment:
In practice it doesn't make sense, but for testing it's completely fine imo.
It should act just like models.Model would do.